tracer/src/Datadog.Trace.MSBuild/DatadogLogger.cs

```diff
@@ -174,29 +175,36 @@ private void EventSource_ProjectStarted(object sender, ProjectStartedEventArgs e
 
                 string projectName = Path.GetFileName(e.ProjectFile);
 
-                Span projectSpan = _tracer.StartSpan(BuildTags.BuildOperationName, parent: parentSpan.Context);
+                string targetName = string.IsNullOrEmpty(e.TargetNames) ? "build" : e.TargetNames?.ToLowerInvariant();
```

Comment:
Could this contain more than one target name? How should we handle that?

Reply:
I haven't seen that case, I guess it needs more testing. Currently I have only test it in our solution and simple samples.